### PR TITLE
AP-1640 Remove feature flag code for non-passported applications

### PR DESCRIPTION
**Pull request by eloisetait3**
**What**

[Link to story](https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=233&projectKey=AP&modal=detail&selectedIssue=AP-1640&assignee=5f968777632c6200712e443f)

Feature flag code is removed for non-passported, including the flag mechanism on the admin/settings page. This is so providers with non-passported rights can progress past DWP checks.

**Checklist**

Before you ask people to review this PR:

- [x] Tests and rubocop should be passing: `bundle exec rake`
- [x] Github should not be reporting conflicts; you should have recently run `git rebase master`.
- [x] There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- [x] The PR description should say what you changed and why, with a link to the JIRA story.
- [x] You should have looked at the diff against master and ensured that nothing unexpected is included in your changes.
- [x] You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: features/step_definitions/civil_journey_steps.rb [hunk 1]
```diff
@@ -707,11 +707,9 @@
 # rubocop:enable Lint/Debugger
 
 Given('I start the passported journey') do
-  LaaApplyForLegalAid::Application.config.x.allow_non_passported_route = false
```
Comment (colinbruce) [inline features/step_definitions/civil_journey_steps.rb:710]: If the step is empty it should probably be removed completely

File: features/step_definitions/civil_journey_steps.rb [hunk 2]
```diff
@@ -707,11 +707,9 @@
...
 end
 
 Then('I complete the passported journey') do
-  LaaApplyForLegalAid::Application.config.x.allow_non_passported_route = true
```
Comment (colinbruce) [inline features/step_definitions/civil_journey_steps.rb:714]: If the step is empty it should probably be removed completely too, along with references to it in the features

**Review by colinbruce — CHANGES_REQUESTED** [review]
Happy to discuss, but some changes may be needed?

File: features/step_definitions/civil_journey_steps.rb (hunk 2 above)
Comment (eloisetait3) [reply features/step_definitions/civil_journey_steps.rb:714]: Yep can do! Will removing it only affect Cucumber if feature references aren't removed?

Comment (colinbruce) [reply features/step_definitions/civil_journey_steps.rb:714]: It should do! :)

**Review by colinbruce — APPROVED** [review]
Nice :+1

**Pull request merged**